libraries/core_libs/network/include/network/tarcap/packets_handlers/get_dag_sync_packet_handler.hpp

```diff
@@ -20,7 +20,8 @@ class GetDagSyncPacketHandler : public PacketHandler {
 
   virtual ~GetDagSyncPacketHandler() = default;
 
-  void sendBlocks(dev::p2p::NodeID const& peer_id, std::vector<std::shared_ptr<DagBlock>> blocks);
+  void sendBlocks(dev::p2p::NodeID const& peer_id, std::vector<std::shared_ptr<DagBlock>> blocks,
```

Comment:
std::vector<std::shared_ptr<DagBlock>> should be probably `const ref` - a lot of data has to be copied here